lib/checksrc.pl

```diff
@@ -501,7 +501,7 @@ sub scanfile {
         if($l =~ /^(.*\W)
                    (gets|
                     strtok|
-                    v?sprintf|
+                    v?sprintf|snprintf|vsnprintf|
```

Comment:
Can we get away with `v?sn?printf` here? Or is that too unreadable even if it works perhaps..

Reply:
I figured that's a bit hard on the eyes. Maybe a middle-ground: `v?sprintf|v?snprintf|` ?